lib/spoom/deadcode/indexer.rb

```diff
@@ -174,6 +174,18 @@ def visit_constant_read_node(node)
         reference_constant(node.name.to_s, node)
       end
 
+      sig { override.params(node: Prism::ConstantPathNode).void }
+      def visit_constant_path_node(node)
+        # In prism 0.28.0, ConstantPathNode was updated to have no children
+        # and instead store the constant name as a string.
+        if node.respond_to?(:name)
+          visit(node.parent) if node.parent
+          reference_constant(node.name.to_s, node)
+        end
+
+        super
+      end
```

Comment:
The [prism 0.28.0 release](https://github.com/ruby/prism/releases/tag/v0.28.0) changed the implementation of `ConstantPathNode` so that it no longer has any children and instead just stores the name of the constant as a string. I added this new visitor to handle that case. I wonder if there's a more elegant way to write this?

Reply:
This seems fine, I can't think of a another way.